app/javascript/gamesFactory.js

```diff
@@ -74,15 +75,19 @@ angular.module('dappChess').factory('games', function (crypto, navigation,
    *  },
    *  ended: <boolean>,
    *  pot: <number>,
+   *  turnTime: <number>,
    *  timeoutStarted: <date>,
-   *  timeoutState: <{-2,-1,0,1,2}>
+   *  timeoutState: <{-1,0,1}>
```

Comment:
No, must be a merge mistake. I took that line from @Kyroy @HeavyPanda 's branch, so I thought it should be right